HadoopWindowsUserSetup.md

```diff
@@ -0,0 +1,130 @@
+****Prerequisites:****
+Before you start installing Hadoop on Windows, there are a few prerequisites that you need to have in place:
+
+1. Java Development Kit (JDK) version 8 or higher
+2. Apache Hadoop distribution suitable for Windows
+
+**Step 1:**  *Install the Java Development Kit* 
+`Hadoop is built using Java, so you’ll need to install the Java Development Kit (JDK) version 8 
+or higher on your computer. You can download the JDK from the Oracle website.
+(https://www.oracle.com/in/java/technologies/javase/javase8-archive-downloads.html) Once the download is 
+complete, run the installer and follow the instructions to install the JDK.`
+
+**Step 2:** *Download the Hadoop distribution*
+`To install Hadoop on Windows, you’ll need to download the appropriate distribution from the 
+Apache Hadoop website (https://hadoop.apache.org/releases.html). 
+You’ll want to choose the distribution that is compatible with your version of Windows (hadoop-3.3.6) and click on binary. 
+Once you’ve downloaded the distribution, extract files of hadoop-3.3.6.tar.gz and place under “C:\Hadoop”.`
+
+**Step 3:** *Set up the Environment Variables*
+
+To use Java & Hadoop, you’ll need to set up some environment variables.
+This will allow you to run Java & Hadoop commands from any directory on your computer. 
+To set up the environment variables, follow these steps:
+
+1. Open the Start menu and search for “Environment Variables”.
+2. Click on “Edit the system environment variables”.
+3. Click on the “Environment Variables” button.
+4. Under “System Variables”, click on “New”.
+5. Enter “JAVA_HOME” as the variable name & the path to the directory where your java is installed (example- C:\Program Files\Java\jdk1.8.0) as the variable value.
+6. Click “OK”.
+7. Enter “HADOOP_HOME” as the variable name and the path to the directory where you extracted the Hadoop distribution (example- C:\hadoop) as the variable value.
+8. Click “OK”.
+9. Locate the “Path” variable in the “System Variables” list and click “Edit”.
+10. Add the following to the end of the “Variable value” field: %JAVA_HOME%\bin; %HADOOP_HOME%\bin; %HADOOP_HOME%\sbin;
+11. Click “OK” to close all the windows.
+
+**Step 4:** *Install Hadoop native IO binary*
+`Clone or download the winutils repository (https://github.com/cdarlint/winutils/tree/master/hadoop-3.3.5/bin) 
+and copy the contents of hadoop-3.3.5/bin into the extracted location of the Hadoop binary package. 
+In our example, it will be C:\hadoop\bin.`
+
+**Step 5:** *Hadoop Configuration*
```

Comment:
You can mention that this step is not necessary for Spark to run, only the winutils and environment variables

Reply:
Done, mentioned it as an important note.